### PR TITLE
Validation of Tutorials 1 & 2

### PR DESCRIPTION
**Pull request by jessiemongeon1**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [x] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [x] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

This is the first PR in my project involving validating the accuracy of each documentation page. This PR covers Tutorials 1&2 in the 'Tutorials' section of the developer docs. Edits to these pages include adding additional context for some concepts such as cycles wallets, adding a 'Next steps' section to pages that did not previously have it, and some minor rewording for clarity.

### CODE REVIEW

File: docs/tutorials/02_create_your_first_app/05_storing-data.md [hunk 1]
```diff
@@ -74,13 +74,14 @@ Here's the code with comments inline:
     };
 
 ```
+
 ### What this does
-- The `getVotes` method returns an `Array`. [`Array`](../../motoko/main/base/Array.md) is a simple container that can hold multiple values.
-- In this particular case, the array holds elements that is a tuple. The tuples in this case are of type `(Text, Nat)`
-- We use a class `Iter` that represents an iterator, a pointer-like data structure that allows the developer to move over another data structure and see values one by one in a sequential manner.
+- The `getVotes` method returns an `Array`. [`An array`](../../motoko/main/base/Array.md) is a simple container that can hold multiple values.
```
Comment (dprats) [inline docs/tutorials/02_create_your_first_app/05_storing-data.md:79]: [`An array`] looks odd to my eyes but not opinionated

**Comment by dprats** [comment]
just one comment, approved otherwise

**Review by dprats — APPROVED** [review]

File: docs/tutorials/02_create_your_first_app/05_storing-data.md (hunk 1 above)
Comment (jessiemongeon1) [reply docs/tutorials/02_create_your_first_app/05_storing-data.md]: Good call! I removed the redundant 'an array' bit and combined the two sentences into one so its a little better on the eyes.

**Comment by aubort** [comment]
I've added a comment in the code. If the PR is already closed and merged, I can open a separate one.

**Comment by jessiemongeon1** [comment]
> I've added a comment in the code. If the PR is already closed and merged, I can open a separate one.

Hey Pascal, please open a new PR for the edit. Thank you!

**Comment by aubort** [comment]
Thanks @jessiemongeon1. I've created the PR here: https://github.com/dfinity/portal/pull/1526

**Pull request merged**